src/services/contract-service.js

```diff
@@ -13,6 +13,7 @@ import bs58 from 'bs58'
 import Web3 from 'web3'
 
 const emptyAddress = '0x0000000000000000000000000000000000000000'
+const NUMBER_CONFIRMATIONS_TO_REPORT = 20
```

Comment:
Is 20 a bit too high ? I thought typically 6 to 10 confirmations is good ? Maybe we pick 10 ?

Reply:
Ok. Sg. Let's keep 20 for now then. Thanks for the details :)

Reply:
My reason for 1️⃣:two:: https://ethereum.stackexchange.com/a/3009/42177

Reply:
Thanks for explanation @micahalcorn